low-level/files/opt/seagate/sspl/bin/sspl_config

```diff
@@ -185,15 +185,28 @@ case $cmd in
         ;;
 esac
 
+rabbitmq_version=$(rpm -qi rabbitmq-server | awk -F': ' '/Version/ {print $2}')
+rabbitmq_base_version=$(echo "${rabbitmq_version:0:-3}" | sed 's/.$//')
+rabbitmq_updated_version=$(echo "${rabbitmq_version::-1}" | sed 's/.$//')
+rabbitmq_middle_version=$(echo "${rabbitmq_updated_version:2:3}")
+
 # Skip this step if sspl is being configured for node replacement scenario as consul data is already
 # available on healthy node
 # Updating RabbitMQ cluster nodes.
 # In node replacement scenario, avoiding feeding again to avoid over writing already configured values
 # with which rabbitmq cluster may have been created
 [ -f $REPLACEMENT_NODE_ENV_VAR_FILE ] || {
     # Update cluster_nodes key in consul
-    out=$(rabbitmqctl cluster_status | grep running_nodes | cut -d '[' -f2 | cut -d ']' -f1 | sed 's/rabbit@//g' | sed 's/,/, /g')
-    pout=$(echo $out | sed  "s/'//g" | sed  "s/ //g")
+    if [ $rabbitmq_base_version -eq 3 ] && [ $rabbitmq_middle_version -ge 8 ];
```

Comment:
[ $rabbitmq_middle_version -eq 8 ]

would be  better, so we dont know if minor version 9 would support same format or not. if we upgrade

Reply:
Done